v0.0.1/model-schema.json

```
@@ -0,0 +1,192 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/model-schema.json",
+  "title": "ForecastHub model metadata",
```

Suggesting new title
```suggestion
  "title": "Schema for Modeling Hub model metadata",
```